code/modules/mod/modules/module_kinesis.dm

```diff
@@ -248,3 +253,13 @@
 	given_turf = locate(mob_x+our_x-round(view[1]/2),mob_y+our_y-round(view[2]/2),mob_z)
 	given_x = round(icon_x - world.icon_size * our_x)
 	given_y = round(icon_y - world.icon_size * our_y)
+
+/obj/item/mod/module/anomaly_locked/kinesis/chrono
+	name = "MOD kinesis+ module" //Chrono Legionarrie variant. You know why, deep in your heart.
```

Comment:
legionnaire
would be better if you called it advanced or kinesis plus

Reply:
💀 
I mean fair, I just don't want a mapper accidentally putting it somewhere it shouldn't be. Oh well, that's a bridge I'll cross another day.